script.js

```diff
@@ -1,4 +1,195 @@
+      "<br>" +
+      "So far,you've been drawing " +
+      DrawCounter +
+      "/" +
+      TotalTurns +
+      " turns.");
+  }
+  // Paper lose to scissors
+  if (input == "paper" && ComputerChoice == "scissors") {
+    LoseCounter = LoseCounter + 1;
+    TotalTurns = TotalTurns + 1;
+    return (myOutputValue =
+      "Computer chose " +
+      ComputerChoice +
+      "." +
+      "<br>" +
+      "You chose " +
+      input +
+      "." +
+      "<br>" +
+      "Sorry but you lose!" +
+      "<br>" +
+      "So far,you've been losing " +
+      LoseCounter +
+      "/" +
+      TotalTurns +
+      " turns.");
+  }
+  // Paper win stone
+  if (input == "paper" && ComputerChoice == "stone") {
+    WinCounter = WinCounter + 1;
+    TotalTurns = TotalTurns + 1;
+    return (myOutputValue =
+      "Computer chose " +
+      ComputerChoice +
+      "." +
+      "<br>" +
+      "You chose " +
+      input +
+      "<br>" +
+      "You win!" +
+      "<br>" +
+      "So far,you've been winning " +
+      WinCounter +
+      "/" +
+      TotalTurns +
+      " turns.");
+  }
+  // Paper draw paper
+  if (input == "paper" && ComputerChoice == "paper") {
+    DrawCounter = DrawCounter + 1;
+    TotalTurns = TotalTurns + 1;
+    return (myOutputValue =
+      "Computer chose " +
+      ComputerChoice +
+      "." +
+      "<br>" +
+      "You chose " +
+      input +
+      "." +
+      "<br>" +
+      "Its a draw!" +
+      "<br>" +
+      "So far,you've been drawing " +
+      DrawCounter +
+      "/" +
+      TotalTurns +
+      " turns.");
+  }
+  // Stone lose to paper
+  if (input == "stone" && ComputerChoice == "paper") {
+    LoseCounter = LoseCounter + 1;
+    TotalTurns = TotalTurns + 1;
+    return (myOutputValue =
+      "Computer chose " +
+      ComputerChoice +
+      "." +
+      "<br>" +
+      "You chose " +
+      input +
+      "." +
+      "<br>" +
+      "Sorry but you lose!" +
+      "<br>" +
+      "So far,you've been losing " +
+      LoseCounter +
+      "/" +
+      TotalTurns +
+      " turns.");
+  }
+  // Stone win scissors
+  WinCounter = WinCounter + 1;
+  TotalTurns = TotalTurns + 1;
+  if (input == "stone" && ComputerChoice == "scissors") {
+    return (myOutputValue =
+      "Computer chose " +
+      ComputerChoice +
+      "." +
+      "<br>" +
+      "You chose " +
+      input +
+      "<br>" +
+      "You win!" +
+      "<br>" +
+      "So far,you've been winning " +
+      WinCounter +
+      "/" +
+      TotalTurns +
+      " turns.");
+  }
+  // Stone draw stone
+  if (input == "stone" && ComputerChoice == "stone") {
+    DrawCounter = DrawCounter + 1;
+    TotalTurns = TotalTurns + 1;
+    return (myOutputValue =
+      "Computer chose " +
+      ComputerChoice +
+      "." +
+      "<br>" +
+      "You chose " +
+      input +
+      "." +
+      "<br>" +
+      "Its a draw!" +
+      "<br>" +
+      "So far,you've been drawing " +
+      DrawCounter +
+      "/" +
+      TotalTurns +
+      " turns");
+  }
```

Comment:
using the || would greatly reduce the amount of duplicate code you have 
all the win conditions can be reduced to this:
if (
(input == "stone" && ComputerChoice == "scissors") ||
(input == "scissors" && ComputerChoice == "paper") ||
(input == "paper" && ComputerChoice == "stone") )
{
outputString="You win..... "
}